roaring/README.md

```diff
@@ -0,0 +1,34 @@
+# The Fuzzer
+
+For complete documentation on go-fuzz, please see: https://github.com/dvyukov/go-fuzz
+
+The fuzzer in relation to the roaring package checks the `Bitmap.UnmarshalBinary` function found in `roaring.go`. In order to use the fuzzer, you can follow these steps:
+
+`cd $GOPATH/src/github.com/pilosa/pilosa/roaring`
+
+`go-fuzz-build ./`
+
+You must now make the workdir/corpus directory. This is achieved by:
+
+`mkdir workdir/corpus`
+
+The fuzzer needs some input to start the fuzzing with. Copy some sample Pilosa fragments into the workdir/corpus folder. For example:
+
+`cp ~/.pilosa/my-index/my-field/views/standard/fragments/0 workdir/corpus`
+
+Once you have copied your sample inputs, you are ready to run the fuzzer:
+
+`go-fuzz -bin=roaring-fuzz.zip -workdir=workdir -func=FuzzBitmapUnmarshalBianry`
```

Comment:
typo at `FuzzBitmapUnmarshalBinary`, but maybe we can change this to the more concise `FuzzUnmarshalBinary` or just `FuzzUnmarshal`?

Reply:
There are like 2 UnmarshalBinary functions. Shouldn't it be specific so we remember which function we are fuzzing?

Reply:
I don't think we're gonna be fuzzing the other one anyway, and the context can be seen by looking which function the fuzzer calls in its definition

Reply:
I'd err on the side of being more specific... it really only helps readability in this case, and if you're looking at the test output from a high level, you might not even know what package this is in